src/main/java/ch/njol/skript/expressions/ExprName.java

```diff
@@ -87,24 +91,26 @@
 	"2.4 (non-living entity support, changeable inventory name)",
 	"2.7 (worlds)"
 })
+@Deprecated(since="INSERT VERSION", forRemoval = true)
 public class ExprName extends SimplePropertyExpression<Object, String> {
 
 	@Nullable
 	private static BungeeComponentSerializer serializer;
 
 	static {
-		// Check for Adventure API
-		if (Skript.classExists("net.kyori.adventure.text.Component") &&
-			Skript.methodExists(Bukkit.class, "createInventory", InventoryHolder.class, int.class, Component.class))
-			serializer = BungeeComponentSerializer.get();
+		if (!SkriptConfig.useTypeProperties.value()) {
```

Comment:
Guard class might be better for indentation management

Reply:
if you meant guard clause, I didn't both since this will just be removed in 2 days anyway, once 2.13 is merged.